src/app/app/page.tsx

```diff
@@ -0,0 +1,80 @@
+import { classes, container } from "@/shared/tw";
+
+type Status = "active" | "inactive";
+
+type Role = "admin" | "user";
+
+type User = {
+  id: string;
+  name: string;
+  email: string;
+  status: Status;
+  role: Role;
+}
+
+// list with 6 random Users generated
+const users: User[] = [
```

Comment:
Just to be clear, this is only some placeholder code. Feel free to change it as you need